libs/ml/include/ml/ops/weights.hpp

```diff
@@ -215,6 +215,14 @@ class Weights : public fetch::ml::ops::PlaceHolder<T>, public Trainable<T>
       }
       break;
     }
+    case WeightsInitialisation::ONES:
```

Comment:
this looks very weird.
both zero and ones case should be implemented like this:

tensor.Fill(0);
tensor.Fill(1);